streams/src/main/java/org/apache/kafka/streams/state/internals/MemoryNavigableLRUCache.java

```diff
@@ -31,13 +31,6 @@ public MemoryNavigableLRUCache(String name, final int maxCacheSize, Serde<K> key
         super(name, maxCacheSize, keySerde, valueSerde);
     }
 
-    @Override
-    public MemoryNavigableLRUCache<K, V> whenEldestRemoved(EldestEntryRemovalListener<K, V> listener) {
```

Comment:
This is intentional since the inherited class as the same function, so no need to override.